code/game/gamemodes/dynamic/dynamic_rulesets_midround.dm

```diff
@@ -25,6 +25,8 @@
 	required_type = /mob/dead/observer
 	/// Whether the ruleset should call generate_ruleset_body or not.
 	var/makeBody = TRUE
+	/// The rule needs this many applicants to be properly executed.
+	var/required_applicants = 1
```

Comment:
Move this up to `from_ghosts`, since the base midround ruleset is also used for non-applicant rulesets such as Sleeper Agent.

Reply:
uhh...
But I thought this var was already in `from_ghosts`